recipes/upf_to_json/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "upf_to_json" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e5702ddb121477cf44c7ae4e713b526c26c0326fdaa3ca30fe1fc2d140f5e3ef
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - upf_to_json
+
+about:
+  home: https://github.com/simonpintarelli/upf_to_json
+  license: BSD
+  license_family: BSD
```

Comment:
Please, add the ``license_file``

Reply:
Thanks. I've added the license file. All tests are passing now.